README.md

```diff
@@ -92,5 +92,5 @@ You will also see any lint errors in the console.
 Kyle's favourite dad joke: 
     What time did Sean Connery get to Wimbledon? 
     Tennish. <br />
-Dhruv's favourite dad joke:
+Dhruv's favourite dad joke:asd
```

Comment:
I think you may have accidentally added asd here haha

Reply:
Yeah mb, I made another pull request that I rebased.